src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -907,11 +907,9 @@ function GlobeControls(view, target, radius, options = {}) {
             if (this.enabled === false) return;
 
             event.preventDefault();
-
-            const staticPos = window.getComputedStyle(event.target.parentElement).position !== 'static';
-            const bounds = staticPos ? event.target.getBoundingClientRect() : { left: 0, top: 0 };
-            const x = event.clientX - event.target.offsetLeft - bounds.left;
-            const y = event.clientY - event.target.offsetTop - bounds.top;
+            const coords = this._view.eventToViewCoords(event);
```

Comment:
this actually fixes a bug that is on master: if the page is scrollable, the previous version didn't pick the correct position when the page was scrolled.